tools/test.py

```diff
@@ -91,16 +91,14 @@ def main():
         from mmdet.models import build_detector as build_model
         if 'detector' in cfg.model:
             cfg.model = cfg.model.detector
-    elif cfg.get('USE_MMCLS', False):
-        from mmtrack.apis import multi_gpu_test, single_gpu_test
-        from mmtrack.datasets import build_dataloader
-        from mmtrack.models import build_reid as build_model
-        if 'reid' in cfg.model:
-            cfg.model = cfg.model.reid
     else:
         from mmtrack.apis import multi_gpu_test, single_gpu_test
         from mmtrack.datasets import build_dataloader
-        from mmtrack.models import build_model
+        if cfg.get('TRAIN_REID', False):
+            from mmtrack.models import build_reid as build_model
+            cfg.model = cfg.model.reid
+        else:
+            from mmtrack.models import build_model
```

Comment:
Please attempt to test a reid model to guarantee that there is no bug.
Beside, please directly test Tracktor and DeepSORT to verify the performance